backend/app/core/search.py

```diff
@@ -753,6 +753,7 @@ def create_search_response_document(
         document_category=opensearch_match.document_category,
         document_source_url=opensearch_match.document_source_url,
         document_url=s3_to_cdn_url(opensearch_match.document_url),
+        # TODO - we now need to get this from the db
```

Comment:
Leaving this here for now as a reminder - happy to remove if you think this is not necessary @joel-wright

Reply:
something to talk about rather than deciding now I think